src/dom-to-image.js

```diff
@@ -244,6 +259,86 @@
                         });
                     }
                 }
+
+                if(domtoimage.impl.options.scrollFix &&
+                    (original.scrollTop || original.scrollLeft)) {
+                    // Setup container for absolute positioning of children
+                    clone.style.position = 'relative';
+                    clone.style.overflow = 'hidden';
+                    clone.style.width = original.offsetWidth + 'px';
+                    clone.style.height = original.offsetHeight + 'px';
+                    var scrollTopRemaining = original.scrollTop > 0 ? original.scrollTop : null;
+                    var scrollLeftRemaining = original.scrollLeft > 0 ? original.scrollLeft : null;
+                    var originalIsPositionRelative = originalStyle['position'] === 'relative';
+                    var computedStylesCache = {};
+                    var boundingRectCache = {};
+                    
+                    // Loop through children and set position based on original
+                    // childs position and original containers scroll position
+                    for(var i = 0; i < clone.children.length; i++) {
+                        // Make sure this element is stylable
+                        if(typeof(clone.children[i]) === 'undefined' ||
+                            clone.children[i] === null ||
+                            typeof(clone.children[i].style) === 'undefined') {
+
+                            continue;
+                        }
+                        var originalChildStyles = computedStylesCache[i] || window.getComputedStyle(original.children[i]);
+                        computedStylesCache[i] = originalChildStyles;
+
+                        // Set child to absolute positioning relative to parent (container)
+                        clone.children[i].style.position = 'absolute';
+
+                        // Take into account the fact that there may be children which were already
+                        // positioned absolute relative to its parent, thus we need to use the original position
+                        if (originalIsPositionRelative && originalChildStyles['position'] === 'absolute') {
+                            var top = parseInt(originalChildStyles['top']);
+                            var left = parseInt(originalChildStyles['left']);
+                            var finalTop = isNaN(top) ? 0 : top;
```

Comment:
Don't really need new `var` values for these, since we didn't `const` the original `top` and `left`